client/rpc/src/state/state_full.rs

```diff
@@ -548,60 +522,63 @@ where
 						(key, v)
 					})
 					.collect();
-				vec![Ok(Ok(StorageChangeSet { block, changes }))]
+				vec![StorageChangeSet { block, changes }]
```

Comment:
I _think_ we could skip the `vec!` and just use an array here since arrays impl `IntoIterator` for a while now, not sure how it affects streams though.

Reply:
ty, I could just remove the `Vec` entirely and skip the unwrap_or_default because `Option` impls `IntoIter`